FirebaseAppCheck/Sources/AppAttestProvider/DCAppAttestService+FIRAppAttestService.h

```diff
@@ -15,7 +15,7 @@
  */
 
 // Currently DCAppAttestService is available on iOS only.
-#if TARGET_OS_IOS
+#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
```

Comment:
Maybe not in this PR, but, I think we should allow using App Attest on macOS and Catalyst with Xcode 12.5+. We probably will need to introduce a separate flag like `FIR_APP_ATTEST_SUPPORTED` where we will take into account the Xcode version, platform and OS version (if needed). Then this flag can be used instead of this condition.

Reply:
Adding support in the future sounds good to me. Agreed that it is beyond the scope of this PR that is focused on fixing CI to unblock other work.